Gruntfile.js

```diff
@@ -29,6 +29,14 @@ module.exports = function (grunt) {
       }
     }()),
 
+    i18nextract: {
+      default_options: {
+        src: [ 'src/**/*.js', 'src/**/*.html' ],
+        lang: ['en'],
+        dest: 'build/tmp'
```

Comment:
@jbudz is there a better place to put temporary build files? Ideally someplace that gets cleaned up before/after each run?